docs/hiring-guide/community-spotlight/interview-celeste-horgan.rst

```diff
@@ -0,0 +1,85 @@
+    really enjoy user experience. I enjoy thinking about things from the perspective of the user, but I didn't necessarily enjoy the
+    design part of it. So when I was looking for co-op positions, I saw BlackBerry, the mobile phone company, had an application for a
+    technical writer. So I clicked on the job description, and it was kind of about describing code and helping people use things. And
+    I thought, "Oh, I think I could probably do this." When you're in design school, you typically end up writing briefs or documents
+    based on your designs as well as actually producing the design. So I sent in a few of my design documents as samples. I got an
+    interview, and I more or less learned on the job. I was there for about eight months out in Waterloo, Ontario, working with the
+    BlackBerry team on their Java SDK. 
+    
+    I was one of 600 co-ops they took in that semester. Because they took in so many, they had a
+    pretty rigorous training program in every department. I was one of about 5 or 10 in a 40 person technical writing department. They
+    put us all through a training program where they taught us what DITA was. They taught us about single source authoring. And, yeah,
+    that's kind of where I started. After I graduated university, I worked as a web designer for a year or two, but it was mostly 
+    stubbornness on my part. Because I had done this degree I was like, "No, I'm gonna do it! I'm gonna go be a designer!" But I
+    really didn't like it. My initial thoughts when I was in school were correct. It just wasn't that interesting to me. So I switched
+    back into technical writing in 2016, and that's what I've been doing ever since. I'm at my second job now as a technical writer.
+    Since then I've worked at a company called Elastic Path, in Vancouver. And now I'm in Berlin at a company called commercetools.
+
+**What was it like searching for jobs outside of Canada?**
+
+    You know what? It was almost comically easy. I kind of just dropped my resume in people's inboxes, using the forms on their
+    websites. I wasn't going out of my way to network or anything, and there was a surprising amount of interest. And I was applying
+    quite globally. I applied for a few places in the U. S. and got responses. I think technical writing is kind of a funny of job in
+    that I think technical writers in organizations often feel very undervalued. But in my last job, and in my current job, I've
+    actually done quite a bit of interviewing and quite a bit of hiring. And it's so funny, because you see so many people who just
+    don't seem to have a lot of self-confidence or passion. So if you're even a little bit passionate or a little opinionated, or you
+    kind of just got something going on, you stand out in an interview so quickly. And people are willing to take you abroad for that,
+    which is really cool.
+
+**What is it like working abroad as a technical writer?**
```

Comment:
Abroad feels kinda odd to me personally. Would recommend something like "Outside of your home country" or something.

Reply:
I've made that change. And I like the idea of adding the date, but the index might get a little cluttered with all these dates that are within a few months time span. What do you think about putting a byline underneath the header, and would there be a way to make that text small so it wouldn't look weird?